### ISSUE TITLE
[Module Proposal]: `avm/res/fabric/capacity`

### ISSUE THREAD

**Issue by FlorisWaternet** [Azure/Azure-Verified-Modules#1192]
### Check for previous/existing GitHub issues/module proposals

- [X] I have checked for previous/existing GitHub issues/module proposals.

### Check this module doesn't already exist in the module indexes

- [X] I have checked for that this module doesn't already exist in the module indexes.

### Bicep or Terraform?

Bicep

### Module Classification?

Resource Module

### Module Name

avm/res/fabric/capacity

### Module Details

Microsoft Fabric is a unified platform that can meet your organization's data and analytics needs.
documentation: https://learn.microsoft.com/en-us/fabric/

[MB] Related documentation: https://learn.microsoft.com/en-us/rest/api/microsoftfabric/operation-groups

### Do you want to be the owner of this module?

No

### Module Owner's GitHub Username (handle)

hundredacres

**Comment by ChrisSidebotham** [comment]
Hi @FlorisWaternet - Please can you elaborate on the resource module here? Provide a like to the bicep ref etc?

**Comment by FlorisWaternet** [comment]
@ChrisSidebotham , I'm not if I understand your question correctly. But I did find another issue under the bicep repo in Github Azure, that points to the same question. I don't know what the correct place would be to follow up. But either one would be enough I guess..
https://github.com/Azure/bicep/issues/12905

**Comment by ChrisSidebotham** [comment]
Hi @FlorisWaternet - I am asking the module should consist of, currently bicep only support `Microsoft.Fabric/Capacities` which we can make a resource module. However to support full configuration of Fabric, bicep at this moment does not yet support this

**Comment by hundredacres** [comment]
@ChrisSidebotham I can take on ownership of the module and support the existing Capabilities aspects of the service. We can add support for more features as they get unlocked.

**Comment by hundredacres** [comment]
@matebarabas I can take this one if we determine it's needed and can work on it during Hackathon.

**Comment by matebarabas** [comment]
Hi @hundredacres,

Thanks for requesting/proposing to be an AVM module owner!

We just want to confirm **you agree to the below pages** that define what module ownership means:

- [Team Definitions & RACI](https://azure.github.io/Azure-Verified-Modules/specs/shared/team-definitions)
- [Shared Specification (Bicep & Terraform)](https://azure.github.io/Azure-Verified-Modules/specs/shared)
- [Module Support](https://azure.github.io/Azure-Verified-Modules/help-support/module-support)

Any questions or clarifications needed, let us know!

If you agree, please just **reply to this issue with the exact sentence below** (as this helps with our automation 👍):

"I CONFIRM I WISH TO OWN THIS AVM MODULE AND UNDERSTAND THE REQUIREMENTS AND DEFINITION OF A MODULE OWNER"

Thanks,

The AVM Core Team

#RR

**Comment by hundredacres** [comment]
I CONFIRM I WISH TO OWN THIS AVM MODULE AND UNDERSTAND THE REQUIREMENTS AND DEFINITION OF A MODULE OWNER

**Comment by matebarabas** [comment]
Hi @hundredacres,

Thanks for confirming that you wish to own this AVM module and understand the related requirements and responsibilities!

Before starting development, please ensure ALL the following requirements are met.

**Please use the following values explicitly as provided in the [module index](https://azure.github.io/Azure-Verified-Modules/indexes/) page**:

- For your module:
  - `ModuleName` - for naming your module
  - `TelemetryIdPrefix` - for your module's [telemetry](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-sfr3---category-telemetry---deploymentusage-telemetry)
- For your module's repository:
  - Repo name and folder path are defined in `RepoURL`
  - Create GitHub teams for module owners and contributors and grant them permissions as outlined [here](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only).
  - Grant permissions for the AVM core team and PG teams on your GitHub repo as described [here](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr9---category-contributionsupport---avm--pg-teams-github-repo-permissions).

Check if this module exists in the other IaC language. If so, collaborate with the other owner for consistency. 👍

You can now start the development of this module! ✅ Happy coding! 🎉

**Please respond to this comment and request a review from the AVM core team once your module is ready to be published! Please include a link pointing to your PR, once available. 🙏**

Any further questions or clarifications needed, let us know!

Thanks,

The AVM Core Team

**Comment by hundredacres** [comment]
PR is here: https://github.com/Azure/bicep-registry-modules/pull/3266

**Comment by matebarabas** [comment]
- [ ] Module published. Failed pipeline: https://github.com/Azure/bicep-registry-modules/actions/runs/11269350751 @hundredacres, please let me know when the module was successfully published! Thank you!
- [x] GitHub teams are available and assigned to their parents.
- [x] `CODEOWNERS` file updated.
- [x] Issue template file updated.

**Comment by hundredacres** [comment]
@matebarabas is there anything I'm missing from this in order to close it out?

**Comment by matebarabas** [comment]
Hi @hundredacres, all I was waiting for was a confirmation from you that the pipeline did eventually run successfully. I've just re-checked it and can see that you already have 2 versions published, so we can close this. Thank you for your work!
![Image](https://github.com/user-attachments/assets/19e5506a-597c-41e4-80d8-5873715f91b0)
